src/scss/overrides/_buttons.scss

```diff
@@ -0,0 +1,47 @@
+.btn {
+  border-radius: 0;
+  background: $white;
+  border: 1px solid $developer-blue;
```

Comment:
If you remove the border-color, it will by default be the element color. Then you can take that line out of the mixin too, right? IIRC you need to use the class `btn` with `btn-primary`, etc. anyway.